radio/src/gui/128x64/model_setup.cpp

```diff
@@ -643,37 +783,73 @@ void menuModelSetup(uint8_t event)
 #endif
 
 #if defined(CPUARM)
-      case ITEM_MODEL_EXTERNAL_MODULE_FAILSAFE:
```

Comment:
Clion screwed up the  identation here. This should have only move the lcd_putsLeft after the if ```(IS_MODULE_XJT(moduleIdx)```